yazi-shared/src/fs/cha.rs

```diff
@@ -90,12 +92,52 @@ impl From<Metadata> for Cha {
 	}
 }
 
+impl From<FileType> for Cha {
+	fn from(file_type: FileType) -> Self {
+		let mut kind = ChaKind::empty();
+		let mut permissions = 0;
+
+		if file_type.is_dir() {
+			kind |= ChaKind::DIR;
+
+			#[cfg(unix)]
+			{
+				permissions |= libc::S_IFDIR;
```

Comment:
Why does the permission here only consider `libc::S_IFDIR`? What about the others?

https://github.com/sxyazi/yazi/blob/afa5936bf5102865ef9d9456823ede8e24ecc42e/yazi-shared/src/fs/fns.rs#L271-L280

Reply:
Thanks for pointing that out! I first thought 'FileType' didn't have that info, but it is there in 'FileTypeExt'.

I will add all the types.

Reply:
Done!